src/forms/yup.js

```diff
@@ -19,6 +19,28 @@ import * as yup from 'yup';
 
 const ARRAY_INDEX_REGEX = /\[([^)]+)\]/;
 
+const hasValidTLD = urlString => {
+  try {
+    const TLD_REGEX = /\.([^.]+)$/;
```

Comment:
issue: this allows any character. TLDs need to end in two or more letters. Perhaps something like `\.[a-z]{2,}$`?